clusterman/autoscaler/autoscaler.py

```diff
@@ -253,11 +253,12 @@ def _compute_target_capacity(self, resource_request: SignalResourceRequest) -> f
         cluster_total_resources = self.pool_manager.cluster_connector.get_cluster_total_resources()
         cluster_allocated_resources = self.pool_manager.cluster_connector.get_cluster_allocated_resources()
         non_orphan_fulfilled_capacity = self.pool_manager.non_orphan_fulfilled_capacity
+        determined_setpoint = self.autoscaling_config.setpoint
         logger.info(f'Currently at target_capacity of {current_target_capacity}')
         logger.info(f'Currently non-orphan fulfilled capacity is {non_orphan_fulfilled_capacity}')
         logger.info(f'Current cluster total resources: {cluster_total_resources}')
         logger.info(f'Current cluster allocated resources: {cluster_allocated_resources}')
-
+        logger.info(f'Current determined setpoint: {determined_setpoint}')
```

Comment:
Two small things:
1. I don't think the word "determined" is really necessary here and might confuse people
2. you don't really need to assign to a variable before referencing it -- you could just do `{self.autoscaling_config.setpoint}`.